packages/framework/src/Model/Pricing/Currency/Currency.php

```diff
@@ -19,6 +19,7 @@ class Currency
     public const ROUNDING_TYPE_INTEGER = 'integer';
 
     public const DEFAULT_EXCHANGE_RATE = '1';
+    public const CZK_EXCHANGE_RATE = '0.04';
```

Comment:
Do you think it would be wort to have more precise name of constant like `CZK_EXCHANGE_RATE_TO_EUR`?

Reply:
yes, fixed in d426bbb2a